polyfill/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "temporal",
+  "version": "1.0.0",
+  "description": "TC39 Temporal Proposal Polyfill",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "for FILE in $(find test -type f -name '*.js' | grep -v 'test/node.js'); do node -r ./test/node.js $FILE; done",
+    "build": "rollup -c rollup.config.js"
```

Comment:
this should also have `"prepublish": "npm run build"`

Reply:
[`prepublish` is deprecated in favour of `prepare`](https://docs.npmjs.com/misc/scripts#prepublish-and-prepare)

Reply:
prepare won’t work in older npm versions; prepublish still works in everything.